src/app/models.rs

```diff
@@ -96,6 +100,35 @@ impl PartialEq for AlbumDescription {
 
 impl Eq for AlbumDescription {}
 
+#[derive(Clone, Debug)]
+pub struct AlbumDescriptionInfo {
+    pub description: AlbumDescription,
+    pub info: AlbumInfoRef,
+}
+
+#[derive(Clone, Debug)]
+pub struct AlbumInfoRef {
```

Comment:
a note on naming: those models I suffixed with Ref where meant to act like a "reference" to some resource; for instance ArtistRef would hold an identifier that would allow having a full Artist struct,  and a name to help display that reference

In that case, maybe AlbumReleaseDetails?